templates/smtp-smarthost/smtp-deployment.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     {{- include "georchestra.labels" . | nindent 4 }}
     app.kubernetes.io/component: {{ include "georchestra.fullname" . }}-smtp
 spec:
-  replicas: 1
+  replicas: {{ .Values.georchestra.smtp_smarthost.replicaCount }}
```

Comment:
I think we are missing the same "shortcut helm": `$webapp` like on the other webapps.

Could you add it so that the whole manifest file use it?

Reply:
Yes, I know. It's also on templates/geonetwork/kibana/kibana-deployment.yaml

To be clear, you'd like me to use $webapp there too, am I right ? 

That will mean changing lines that are absolutely not related with this PR. I'd rather do a distinct PR for this, it would me more clear.

Reply:
we can do it in another PR yes